src/mesh/mechanical/solid/boundary/NonFollowerLoad.cpp

```diff
@@ -44,8 +44,8 @@ NonFollowerLoadBoundary::NonFollowerLoadBoundary(
                                              mesh.connectivities(),
                                              filter_dof_list(3, dof_offset, mesh.connectivities()),
                                              material_coordinates,
-                                             boundary["Time"],
-                                             boundary["Values"][name]);
+                                             boundary,
+                                             std::string(name));
```

Comment:
How come you didn't just use `name` directly and constructed a string from it?

Reply:
because I'm keeping both constructors and they have the same number of parameters. With old json I used to get a compiler error but with the new one, it seems to work.